lms/envs/aws.py

```diff
@@ -424,6 +424,11 @@
 
 GRADES_DOWNLOAD = ENV_TOKENS.get("GRADES_DOWNLOAD", GRADES_DOWNLOAD)
 
+###### CORS ###########
+
+CORS_ORIGIN_WHITELIST = AUTH_TOKENS.get('CORS_ORIGIN_WHITELIST', CORS_ORIGIN_WHITELIST)
+CORS_ORIGIN_ALLOW_ALL = AUTH_TOKENS.get('CORS_ORIGIN_ALLOW_ALL', CORS_ORIGIN_ALLOW_ALL)
```

Comment:
An origin is whitelisted if it's included explicitly in `CORS_ORIGIN_WHITELIST` or if it matches an expression in `CORS_ORIGIN_REGEX_WHITELIST`.  Happy to change this to just `CORS_ORIGIN_REGEX_WHITELIST` if that's simpler.